routemaster/models/client_token.rb

```diff
@@ -0,0 +1,37 @@
+require 'routemaster/models'
+require 'routemaster/mixins/redis'
+require 'securerandom'
+
+module Routemaster
+  module Models
+    class ClientToken
+      include Mixins::Redis
+
+      PREFIX = "api_keys:"
+
+      def self.get_all
+        key_dump = {}
+        existing_keys = _redis.keys "#{PREFIX}*"
```

Comment:
In this case, I think you could use a Redis set as a data structure — then `SMEMBERS` here, `SADD` / `SREM` when adding/removing keys.